libs/device/models/src/lib/overview/overview-config.ts

```diff
@@ -58,6 +58,31 @@ export const DataSyncTileConfigValidator = z.object({
 
 export type DataSyncTileConfig = z.infer<typeof DataSyncTileConfigValidator>
 
+// TODO: Implement proper validation
+export const BackupTileConfigValidator = z.object({
+  type: z.literal("mc-overview-backup"),
+  title: z.string().min(1),
+  dataKey: z.string().min(1),
+  backupFeatures: z
+    .array(
+      z.object({
+        label: z.string().min(1),
+        key: z.string().min(1),
+      })
+    )
+    .optional(),
+  restoreFeatures: z
+    .array(
+      z.object({
+        label: z.string().min(1),
+        keys: z.array(z.string()),
```

Comment:
`keys: z.array(z.string().min(1),`

Reply:
Improved in [c9ddceb](https://github.com/mudita/mudita-center/pull/1725/commits/c9ddcebdc6188ebdd3f4a78dca435c55f2e91f94)